### DISCUSSION TITLE
WordPress Comment Bot

### DISCUSSION THREAD

**Discussion by v4r1able** [community/community#37887] [Repositories]
I coded bot that posts comments on the posts of WordPress based system

View: [https://github.com/v4r1able/wordpress-comment-bot](https://github.com/v4r1able/wordpress-comment-bot)

I specified examples and trials in repo

Simple example:
```php
$singlecomment = \wpcomment\bot::sendComment("https://leventemre.com/test/example-post/", array(
    "comment" => "test comment",
    "author" => "v4r1able",
    "email" => "test@example.com",
    "site_address" => "example.com"
));
```

Don't forget to star if you like it :)

**Comment by CharlesRakesh** [comment]
https://github.com/CharlesRakesh/Elementor-Pro-Form/issues/1
